pyproject.toml

```diff
@@ -368,13 +372,53 @@ combine-as-imports = true
 "tests/providers/google/cloud/operators/test_vertex_ai.py" = ["E402"]
 "tests/providers/google/cloud/operators/vertex_ai/test_generative_model.py" = ["E402"]
 "tests/providers/google/cloud/triggers/test_vertex_ai.py" = ["E402"]
-
 "tests/providers/openai/hooks/test_openai.py" = ["E402"]
 "tests/providers/openai/operators/test_openai.py" = ["E402"]
 "tests/providers/qdrant/hooks/test_qdrant.py" = ["E402"]
 "tests/providers/qdrant/operators/test_qdrant.py" = ["E402"]
 "tests/providers/snowflake/operators/test_snowflake_sql.py" = ["E402"]
 
+# All the providers modules which do not follow TRY002 yet
```

Comment:
This is essentially a TODO list, right?  Or have all possible fixes already been made and these are the unfixable ones?

Reply:
Yep, this is a TODO list, which uses into the previous tasks like these:
- https://github.com/apache/airflow/issues/38270
- https://github.com/apache/airflow/issues/37256
- https://github.com/apache/airflow/issues/37897

Reply:
All of them are fixable, just disable check in this files and prevent to add it into the others.